masterlist.yaml

```diff
@@ -17344,16 +17344,26 @@ plugins:
       - <<: *patchProvided
         subs: [ 'Arena - An Encounter Zone Overhaul' ]
         condition: 'active("Arena.esp") and not active("DeadMansRefugeArenaPatch.esp")'
+    tag: [ C.Location ]
     clean:
+      # version: 1.1.1
       - crc: 0x683B8666
         util: 'SSEEdit v4.0.4'
+      # version: 1.1.2
+      - crc: 0x2BBD7258
+        util: 'SSEEdit v4.0.4'
   - name: 'DeadMansRefugeArenaPatch.esp'
     url:
       - link: 'https://bethesda.net/en/mods/skyrim/mod-detail/4283327/'
         name: 'Dead Man''s Refuge - Arena Patch (Bethesda.net)'
+    tag: [ C.Encounter ]
```

Comment:
**C.Encounter** is for the encounter zone (XEZN) subrecord of cell records. The patch edits an encounter zone record, so the tag is not appropriate here. (In fact, there currently aren't any tags for the encounter zone record.)

```suggestion
```